apps/src/lib/node/ledger/storage/rocksdb.rs

```diff
@@ -37,24 +36,31 @@ pub struct RocksDB(rocksdb::DB);
 
 /// Open RocksDB for the DB
 pub fn open(path: impl AsRef<Path>) -> Result<RocksDB> {
+    #[cfg(unix)]
     let max_open_files = match increase_nofile_limit() {
         Ok(max_open_files) => Some(max_open_files),
         Err(err) => {
             tracing::error!("Failed to increase NOFILE limit: {}", err);
             None
         }
     };
+
     let mut cf_opts = Options::default();
     // ! recommended initial setup https://github.com/facebook/rocksdb/wiki/Setup-Options-and-Basic-Tuning#other-general-options
     cf_opts.set_level_compaction_dynamic_level_bytes(true);
     // compactions + flushes
     cf_opts.set_max_background_jobs(6);
     cf_opts.set_bytes_per_sync(1048576);
+
+    #[cfg(unix)]
+    use std::convert::TryInto;
```

Comment:
Is it better to put `use` at the top of the file?

Reply:
@yito88 I personally have no preferences (since I am new in rust). Is the preferred coding style to always put imports on top of the file?

Reply:
Looking back at the changes, I think it may actually be nicer to move the unix only logic into an inlined sub-module, so we don't have the `#[cfg(unix)]` in so many places. I'll push another commit for it

Reply:
updated in e284c9b